spec/support/jruby_binstubs/bundle

```diff
@@ -0,0 +1,25 @@
+require "rubygems"
+gem "bundler"
+require "bundler"
+require "bundler/fetcher"
+require_relative "../in_process_exec.rb"
+
+module Bundler
+  #:nodoc:
+  class Fetcher
+    # The Bundler user_agent uses SecureRandom, which causes the specs
+    # to run out of entropy and run a lot longer than they need to.
+    def user_agent
+      "gemstash spec"
+    end
+  end
+end
```

Comment:
I don't think it's necessary, this is only in use for JRuby to speed up the build there. MRI doesn't seem to have the same problem, and the random number being generated doesn't really have any benefit in our specs. André pointed out in chat, it's for matching requests, which we don't really care about in the specs (nothing is hitting the actual rubygems servers).